### PR TITLE
Add paragraphs module

### LINKED ISSUE #9: Add Paragraphs module
Since we are targeting Paragraphs in Draft theme I think it's worthwhile including this module.

### PR DESCRIPTION
**Pull request by ozin7**
Closes #9

### CODE REVIEW

**Review by Crazylord — DISMISSED** [review]

**Review by T2L — CHANGES_REQUESTED** [review]
1. Do not put Resolve/Close etc into commit message! Add a message into PR itself and it will close the corresponding ticket(s). See #18. 
1. Follow commit message structure: `#XX Title of the commit`, no colon, no full stop in the end

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ## Draft 2.x.x
+- [Issue 9](https://github.com/lemberg/draft/issues/9) - Add Paragraphs module.
```
Comment (T2L) [inline CHANGELOG.md]: Remove full stop in the end

File: draft.install [hunk 2]
```diff
@@ -42,6 +43,13 @@ function draft_update_8103(&$sandbox = NULL) {
   \Drupal::service('module_installer')->install(['admin_toolbar_links_access_filter']);
 }
 
+/**
+ * Enable Paragraphs module.
+ */
+function draft_update_8601(&$sandbox = NULL) {
+  \Drupal::service('module_installer')->uninstall(['paragraphs']);
```
Comment (T2L) [inline draft.install]: Probaby `\Drupal::service('module_installer')->install(['paragraphs']);`, but there is no need to write updates at this point inho

**Review by T2L — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Crazylord) [reply CHANGELOG.md]: Fixed

File: draft.install (hunk 2 above)
Comment (Crazylord) [reply draft.install]: Removed unnecessary update function

**Review by T2L — APPROVED** [review]

**Pull request merged**